packages/console/src/console-history/ConsoleHistoryItem.scss

```diff
@@ -95,3 +117,37 @@ $button-vert-margin: ConsoleVariables.$button-vert-margin;
     }
   }
 }
+
+.console-history-single-line {
+  top: -0.875rem;
+}
+
+.console-history-two-lines {
+  top: -0.125rem;
+}
+
+.console-history-last-single-line {
+  top: -1.5rem;
+}
+
+.console-history-first-single-line {
+  top: 0;
+}
+
+.console-history-item-contextual-help-content {
+  display: grid;
+  grid-template-columns: auto auto;
+  gap: $spacer-1 $spacer-3;
+  text-align: left;
+  margin-bottom: $spacer-2;
+  white-space: nowrap;
+}
+
+.console-history-item-contextual-help {
+  section[class*='spectrum-ContextualHelp-dialog'] {
```

Comment:
I reckon we just put this in `SpectrumComponent.module.scss` so it applies to all instead of just console history items, since we really do want it to fit the content it all cases.
That can be a separate ticket I guess though.